frontend/src/types/MetricsOptions.ts

```diff
@@ -17,7 +17,7 @@ export interface DashboardQuery extends MetricsQuery {
   additionalLabels?: string;
   workload?: string;
   workloadType?: string;
-  cluster?: string;
+  clusterName?: string;
```

Comment:
Don't change MetricsOptions file, this is another issue (https://github.com/kiali/kiali/issues/6753). What you have to do is to remove clusterName attribute from DashboardQuery type.

Reply:
Got it.
Done.